### PR TITLE
docs: internal docs migration

### PR DESCRIPTION
**Pull request by SiTaggart**
Migrate from a sporadic collection of Gdocs, and Confluence pages to Markdown pages in the monorepo.

Single location, located where we do most of our work and align around our workflows.

TODO:
- [x] technologies
- [x] testing and linting
- [x] third party tools
- [x] research strategy
- [x] committee overview
- [x] retro format
- [x] jira conventions
- [x] epic template

### CODE REVIEW

**Comment by serifluous** [comment]
heart this PR

File: internal-docs/engineering/ci.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# CI/CD
```
Comment (SiTaggart) [inline internal-docs/engineering/ci.md:1]: This file has had some updates made

File: internal-docs/engineering/core/component-rules.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Component rules [WIP]
```
Comment (SiTaggart) [inline internal-docs/engineering/core/component-rules.md:1]: Long lost component rules doc

File: internal-docs/engineering/developer-workflow.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
+# Developer Workflow
```
Comment (SiTaggart) [inline internal-docs/engineering/developer-workflow.md:1]: Some refresh and additions here

File: internal-docs/engineering/developing-locally.md [hunk 4]
```diff
@@ -0,0 +1,352 @@
+# Developing locally
```
Comment (SiTaggart) [inline internal-docs/engineering/developing-locally.md:1]: Minor refresh

File: internal-docs/engineering/hosting-and-deployment.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Hosting and Deployment
```
Comment (SiTaggart) [inline internal-docs/engineering/hosting-and-deployment.md:1]: minor updates

File: internal-docs/engineering/testing.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
+# Testing
```
Comment (SiTaggart) [inline internal-docs/engineering/testing.md:1]: minor addition of applitools

File: internal-docs/engineering/third-party-tool.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# Third party tools
```
Comment (SiTaggart) [inline internal-docs/engineering/third-party-tool.md:1]: applitools addition

File: internal-docs/process/definition-of-done.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# Definition of Done
```
Comment (SiTaggart) [inline internal-docs/process/definition-of-done.md:1]: Added a couple of the items from working agreements update to the components DoD, rest is the same

File: internal-docs/process/definition-of-ready.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+# Definition of Ready
```
Comment (SiTaggart) [inline internal-docs/process/definition-of-ready.md:1]: Some additions from the working agreements session, mostly the same

File: internal-docs/process/design-system-committee.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
+# Design systems committee
```
Comment (SiTaggart) [inline internal-docs/process/design-system-committee.md:1]: Cut down to just the process outline

File: internal-docs/process/how-we-work.md [hunk 11]
```diff
@@ -0,0 +1,165 @@
+# How we work
```
Comment (SiTaggart) [inline internal-docs/process/how-we-work.md:1]: Updated meetings and rituals

File: internal-docs/process/ticketing-conventions.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
+# Ticketing conventions
```
Comment (SiTaggart) [inline internal-docs/process/ticketing-conventions.md:1]: Added ticket details section, expanded pointing

**Review by TheSisb — APPROVED** [review]
Great start, thanks for doing this!

**Review by nkrantz — APPROVED** [review]
Thank you for doing this!

**Pull request merged**